pkgs/development/python-modules/NSKeyedUnArchiver/default.nix

```diff
@@ -0,0 +1,26 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "NSKeyedUnArchiver";
+  version = "1.2";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-OZWPGH8ggYYKxeg6CGyft6ZM9Yx/43z7YylzXlQcTVg=";
+  };
+
```

Comment:
Would be better to add `pythonImportsCheck` for both modules, see https://nixos.org/manual/nixpkgs/unstable/#using-pythonimportscheck